src/index.ts

```diff
@@ -47,14 +47,21 @@ const handleMessage = (message: any) => {
       if (message.state === Connection.CONNECTED) {
         machine.send({ type: 'NEW_BLOCK' });
       }
+      if (message.state === Connection.CONNECTING) {
+        logger.error(`Websocket is attempting to connect to ${process.env.DEFAULT_SERVER}`);
```

Comment:
This is not always an error, the connection process always start with connecting before being connected. Not sure if this can be bad for your debug but it's semantically wrong to this all CONNECTING messages as errors.

Reply:
I agree! Fixed in 39c81a146d0d8e69c1cafa09e4cd5cdbeb34d882